development/compiling/compiling_with_mono.rst

```diff
@@ -58,9 +58,9 @@ Generate the glue
 Glue sources are the wrapper functions that will be called by managed methods.
 These source files must be generated before building your final binaries. In
 order to generate them, first, you must build a temporary Godot binary with the
-options ``tools=yes`` and ``mono_glue=no``::
+options ``target=editor`` and ``mono_glue=no``::
 
-    scons p=<platform> tools=yes module_mono_enabled=yes mono_glue=no
+    scons p=<platform> target=editor module_mono_enabled=yes mono_glue=no
```

Comment:
`mono_glue` is no longer used. Build process is:
- Build executable with `module_mono_enabled=yes`
- Run ` --headless --generate-mono-glue modules/mono/glue`
- Run `./modules/mono/build_scripts/build_assemblies.py --godot-output-dir=./bin --godot-platform={PLATFORM_NAME}`

Reply:
does the first binary need to be a `target=editor` build or can it be one of the `template_debug` or `template_release` templates as well?

Do we need special steps to create `target=template_release production=yes` builds?

In the current documentation examples list all the commands to create all three variants. What would be the updated equivalent?

```sh
    # Build temporary binary
    scons p=windows target=editor module_mono_enabled=yes
    # Generate glue sources
    bin\godot.windows.editor.x86_64.mono --generate-mono-glue modules/mono/glue
    # Build binaries
    ./modules/mono/build_scripts/build_assemblies.py --godot-output-dir=./bin --godot-platform={PLATFORM_NAME}

    # are the below steps needed with the new commands?
    ### Build binaries normally
    # Editor
    scons p=windows target=editor module_mono_enabled=yes
    # Export templates
    scons p=windows target=template_debug module_mono_enabled=yes
    scons p=windows target=template_release module_mono_enabled=yes
```